pkg/patch/patch.go

```diff
@@ -42,6 +42,32 @@ const (
 	defaultTag              = "latest"
 )
 
+type TrivyOpts struct {
+	Image                     string
+	Ch                        chan error
```

Comment:
I'm not sure if it makes sense to have channel in TrivyOpts. Also, so we aren't making this specific to Trivy, we could call it scannerOpts? We would still want this to be compatible with https://github.com/project-copacetic/scanner-plugin-template